src/scss/components/_hero-section.scss

```diff
@@ -73,7 +73,18 @@
   &__cloud {
     filter: $svg-cloud-shadow;
   }
-}
+
+  &__cloud-animation-right {
+    animation: right 200s linear infinite;
+  	}
```

Comment:
formatting

Reply:
@zastrow when you say formatting are you talking about indentation?